# HG changeset patch
# User Rik <rik@octave.org>
# Date 1376601440 25200
#      Thu Aug 15 14:17:20 2013 -0700
# Node ID ce55a85758742e9fef1d0758273f66e04bb56f2e
# Parent  ee1d19174316b38e9487eec84e770ca09c8c27b5
Fix typo in titles for contour/contourf demos.

* scripts/plot/contour.m, scripts/plot/contourf.m: Add missing '}' for
multi-line titles.

diff --git a/scripts/plot/contour.m b/scripts/plot/contour.m
--- a/scripts/plot/contour.m
+++ b/scripts/plot/contour.m
@@ -86,26 +86,26 @@ endfunction
 
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! [x, y, z] = peaks ();
 %! contour (x, y, z);
 %! title ('contour() plot of peaks() function');
-%! title ({'contour() plot (isolines of constant Z)'; 'Z = peaks()');
+%! title ({'contour() plot (isolines of constant Z)'; 'Z = peaks()'});
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! [theta, r] = meshgrid (linspace (0,2*pi,64), linspace (0,1,64));
 %! [X, Y] = pol2cart (theta, r);
 %! Z = sin (2*theta) .* (1-r);
 %! contour (X, Y, abs (Z), 10);
-%! title ({'contour() plot'; 'polar fcn Z = sin (2*theta) * (1-r)'});
+%! title ({'contour() plot'; 'polar fcn: Z = sin (2*theta) * (1-r)'});
 
 %!test
 %! hf = figure ("visible", "off");
 %! clf (hf);
 %! unwind_protect
 %!   [x, y, z] = peaks ();
 %!   [c, h] = contour (x, y, z);
 %!   levellist = (-6):6;
diff --git a/scripts/plot/contourf.m b/scripts/plot/contourf.m
--- a/scripts/plot/contourf.m
+++ b/scripts/plot/contourf.m
@@ -87,26 +87,26 @@ function [c, h] = contourf (varargin)
 endfunction
 
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! [x, y, z] = peaks (50);
 %! contourf (x, y, z, -7:9);
-%! title ({'contourf() plot (filled contour lines)'; 'Z = peaks()');
+%! title ({'contourf() plot (filled contour lines)'; 'Z = peaks()'});
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! [theta, r] = meshgrid (linspace (0,2*pi,64), linspace (0,1,64));
 %! [X, Y] = pol2cart (theta, r);
 %! Z = sin (2*theta) .* (1-r);
 %! contourf (X, Y, abs (Z), 10);
-%! title ({'contourf() plot'; 'polar fcn Z = sin (2*theta) * (1-r)'});
+%! title ({'contourf() plot'; 'polar fcn: Z = sin (2*theta) * (1-r)'});
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! x = linspace (-2, 2);
 %! [x, y] = meshgrid (x);
 %! z = sqrt (x.^2 + y.^2) ./ (x.^2 + y.^2 + 1);
 %! contourf (x, y, z, [0.4, 0.4]);
